ex9_ch3.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#step1: import math
+import math
+
+#step2: get two inputs from user on length and width of room
+ceiling_length = int(input("What is the length of the ceiling? "))
+
+ceiling_width = int(input("What is the width of the ceiling? "))
+
+#step3: make calculation to find the area of the ceiling
+ceiling_area = ceiling_length * ceiling_width
+
+#step4: make calculation to find out how many gallons of paint are needed to paint the ceiling's area
+paint_gallons_needed = math.ceil(ceiling_area/350)
```

Comment:
Excellent use of `ceil` here. Do you see where you can use a constant here? :)

Reply:
Yes, something along the lines of 1 paint gallon covering 350 square feet, since the problem told me from the outset that 1 gallon covers 350 square feet. The constant name would be quite long, right?

ONE_GALLON_COVERS_SQFT = 350

Reply:
Exactly! 👏 

I'd personally name the constant `GALLONS_PER_SQ_FT`, but that's nit-picky.